assets/app/view/game/round/stock.rb

```diff
@@ -406,18 +405,17 @@ def render_select_par_slot
           h(:div, children)
         end
 
-        def render_take_loan
-          take_loan = lambda do
-            process_action(Engine::Action::TakeLoan.new(@current_entity, loan: nil))
+        def render_game_buttons
+          buttons = []
+          @step.game_buttons(@current_entity).each_with_index do |game_button, index|
```

Comment:
Is there a reason you can't do 

buttons = @step.game_buttons(@current_entity).map do |game_button|
  on_click = lambda { process_actions(game_buttton[:action]) }
  h(:button, { on: { click: on_click } }, game_button[:description])
end

Reply:
I was able to do that - I'm not sure what I was doing wrong before when attempting the same thing - but clearly something was wrong.